src/server/config/express.js

```diff
@@ -55,6 +56,14 @@ export default (server) => {
     proxyOneQuran.web(req, res);
   });
 
+  server.use('/apple-app-site-association', (req, res) => {
```

Comment:
Looks like a static file(same as manifest.json), why we need route for this? 

https://quran.com/manifest.json